bazel/repositories.bzl

```diff
@@ -216,7 +216,11 @@ def envoy_dependencies(skip_targets = []):
     external_http_archive("bazel_features")
     external_http_archive("bazel_toolchains")
     external_http_archive("bazel_compdb")
-    external_http_archive("envoy_examples")
+    external_http_archive(
+        name = "envoy_examples",
+        patch_args = ["-p1"],
+        patches = ["@envoy//bazel:envoy_examples.patch"],
```

Comment:
This can be removed after https://github.com/envoyproxy/examples/pull/522 is merged.